h_green_lower: 60
h_green_upper: 100
s_green_lower: 100
s_green_upper: 230
v_green_lower: 50
v_green_upper: 100
h_yellow_lower: 20
h_yellow_upper: 40
s_yellow_lower: 120
s_yellow_upper: 210
v_yellow_lower: 90
v_yellow_upper: 255
h_blue_lower: 90
h_blue_upper: 110
s_blue_lower: 210
s_blue_upper: 255
v_blue_lower: 35
v_blue_upper: 180
h_orange_lower: 10
h_orange_upper: 20
s_orange_lower: 80
s_orange_upper: 200
v_orange_lower: 120
v_orange_upper: 200
h_purple_lower: 110
h_purple_upper: 180
s_purple_lower: 80
s_purple_upper: 230
v_purple_lower: 40
v_purple_upper: 100
h_red_lower: 130
h_red_upper: 180
s_red_lower: 180
s_red_upper: 230
v_red_lower: 80
v_red_upper: 180
